crates/oxc_linter/src/snapshots/no_regex_spaces.snap

```diff
@@ -46,28 +53,28 @@ source: crates/oxc_linter/src/tester.rs
   ⚠ eslint(no-regex-spaces): Spaces are hard to count.
    ╭─[no_regex_spaces.tsx:1:15]
  1 │ var foo = /bar   {3}baz/;
-   ·               ───
+   ·               ──
    ╰────
   help: Use a quantifier, e.g. {2}
 
   ⚠ eslint(no-regex-spaces): Spaces are hard to count.
    ╭─[no_regex_spaces.tsx:1:15]
  1 │ var foo = /bar    ?baz/;
-   ·               ────
+   ·               ───
    ╰────
   help: Use a quantifier, e.g. {2}
 
   ⚠ eslint(no-regex-spaces): Spaces are hard to count.
    ╭─[no_regex_spaces.tsx:1:26]
  1 │ var foo = new RegExp('bar   *baz')
-   ·                          ───
```

Comment:
Are these spans incorrect? This smells like a bug in the regex parser.

Reply:
I think it's the opposite: these are actually more accurate now thanks to the parser. `/   ?/` is not equivalent to `/ {3}/`, as the diagnostic might suggest, since the last space is part of the quantifier. it is actually more like `/ {2,3}/` but that is beyond the scope of what the original lint rule targets.